src/argilla/client/sdk/text_classification/api.py

```diff
@@ -58,6 +58,67 @@ def data(
         )
 
 
+def add_dataset_labeling_rule(
+    client: AuthenticatedClient,
+    name: str,
+    rule: LabelingRule,
+) -> Response[Union[LabelingRule, HTTPValidationError, ErrorMessage]]:
+    url = "{}/api/datasets/{name}/TextClassification/labeling/rules".format(
+        client.base_url, name=name
+    )
+
+    response = httpx.post(
+        url=url,
+        json={"query": rule.query, "label": rule.label},
```

Comment:
This won't work for text-classification with multi-label. You should pass also the `rule.labels` field.